spring-boot-project/spring-boot-autoconfigure/build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
 	optional("org.ehcache:ehcache")
 	optional("org.elasticsearch.client:elasticsearch-rest-client")
 	optional("org.elasticsearch.client:elasticsearch-rest-high-level-client")
+	optional("org.elasticsearch.client:elasticsearch-rest-client-sniffer")
```

Comment:
Dependencies are listed alphabetically. This one should be above `elasticsearch-rest-high-level-client`